src/views/Swap.vue

```diff
@@ -552,10 +548,10 @@ export default {
       return this.sendFeeType === FEE_TYPES.BTC
     },
     amountToSendInFiat () {
-      return prettyFiatBalance(this.amount, this.fiatRates[this.asset])
+      return prettyFiatBalance(this.amount, this.fiatRates[this.assetChain])
```

Comment:
I think you may be confusing assetChain and asset 😄  
I will explain externally

Reply:
Fixed now, thanks for explain